invoices.go

```diff
@@ -0,0 +1,94 @@
+package scalingo
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"gopkg.in/errgo.v1"
+)
+
+type InvoicesService interface {
+	ListInvoices(context.Context, PaginationOpts) (Invoices, PaginationMeta, error)
+	InvoiceShow(context.Context, string) (*Invoice, error)
+}
+
+var _ InvoicesService = (*Client)(nil)
+
+const LayoutISO = "2006-01-02"
```

Comment:
This constant name is not right

```suggestion
const BillingMonthDateFormat = "2006-01-02"
```